packages/connect-express/package.json

```diff
@@ -28,11 +29,12 @@
     "node": ">=16.0.0"
   },
   "dependencies": {
-    "@types/express": "^4.17.18",
+    "express": "^4.19.2",
```

Comment:
To run the express-readme.spec.ts, we need express, but it should be a dev-dependency. It wasn't in connect-node-test because it's all dev.

Can you move express to devDependencies?